QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -1398,10 +1601,62 @@ CylTransFunc::CylTransFunc(double coatThick, double halfCylThick):
       //Use same loss angle for coating whether crystal or effective isotropic
       toSI * fourKbToverPi * coatingEnergy * coatLossAngle;
     const double totalNoiseTimesf = subNoiseTimesf + coatNoiseTimesf;
-
+      
     const double subAmpNoiseTimesSqrtf = sqrt(subNoiseTimesf);
     const double coatAmpNoiseTimesSqrtf = sqrt(coatNoiseTimesf);
     const double totalAmpNoiseTimesSqrtf = sqrt(totalNoiseTimesf);
+
+      
+   MultLossCoatingEnergy /= F0*F0;
+   
+   // MultLossCoatingEnergy is defined with the loss angles already.
+   const double MultLossCoatNoiseTimesf = toSI * fourKbToverPi * MultLossCoatingEnergy;
+   const double MultLossCoatNoiseTimesfSqrtf = sqrt(MultLossCoatNoiseTimesf);
+      
+   const double totalNoiseMultLossAngle = MultLossCoatNoiseTimesf + subNoiseTimesf;
+   const double totalNoiseMultLossAngleSqrtf = sqrt(totalNoiseMultLossAngle);
+
+   // Create the dat file that houses the loss angle information.
+   if (Utilities::MPI::this_mpi_process(mpi_communicator) == 0) {
+       std::ofstream dOut("MultipleLossAngles.dat", std::ios::app);
+          
```

Comment:
Output everything to Energy.dat. Let the user choose whether they want different loss angles or all loss angles (bulk/shear, or 11/12/44) the same, but then just print the results to Energy.dat. Test that when all loss angles are the same, you get the same Energy.dat as before your changes.